homeassistant/components/tuya/vacuum.py

```diff
@@ -187,6 +187,6 @@ def set_fan_speed(self, fan_speed: str, **kwargs: Any) -> None:
         """Set fan speed."""
         self._send_command([{"code": DPCode.SUCTION, "value": fan_speed}])
 
-    def send_command(self, command, params=None, **kwargs) -> None:
+    def send_command(self, command: Any, params: Any = None, **kwargs: Any) -> None:
```

Comment:
The command isn't any? Neither are the params.

Reply:
Adjusted!
The `command` parameter follows the one defined in Tuya base. `params` is not used when sending commands to Tuya integration.
Please review.